doc/bt/ZC_MAP_REVEAL_LIST.bt

```diff
@@ -20,4 +20,5 @@ int count;
 if (!checkCompression())
 {
     mapRevealData dataSets[count];
+    float spacer;
```

Comment:
I believe this can't be correct, because based on my latest logs, the second float is part of every entry in `ZC_MAP_REVEAL_LIST`. There's not only one at the end of the list. Though is seems like the second float is always the same as the first.

Reply:
What would you like the variable to be called?

Reply:
The name is not the problem, that it doesn't exist is^^" I'll take care of this.